ui/App/ProjectScreen/Source/CheckoutPatchButton.svelte

```diff
@@ -73,10 +65,12 @@
 <Overlay {expanded} on:hide={hide} style="position: relative;">
   <div class="request-dropdown" hidden={!expanded}>
     <p style="margin-bottom: 0.5rem;">
-      To check out this Patch locally, run this in your terminal:
+      To fetch this patch in your working copy run the following command.
     </p>
     <Copyable name="command" bind:this={copyable}>
-      <p class="typo-text-small-mono instructions">{instructions}</p>
+      <p class="typo-text-small-mono instructions">
+        upstream patch fetch {Patch.handle(patch)}
```

Comment:
The button action is "Checkout", however the command only fetches the data, but doesn't actually check it out. We should either rename the button to "Fetch latest changes" or show the command to also checkout the code.

Reply:
> But this got me thinking whether the command should actually also check it out: upstream patch checkout PATCH. Is this more useful? (Github shows a checkout command in their PRs.)

Wouldn't `upstream patch checkout PATCH` be just an alias for `git checkout PATCH`, or is there more to it?

Reply:
So `upstream patch checkout PATCH` would fetch and check it out, right? I think that makes sense.

Reply:
After discussing this we decided to go ahead with adding the `git checkout` command. This makes it transparent what is happening and is not a lot more complex.